README.md

```diff
@@ -3,12 +3,30 @@
 This repo forks the FreeRDP repo, thus allowing us to make changes that fit our needs and augment the codebase with other components.
 From time to time there is a need to merge the changes from the original repo into this one.
 
-### ❗When updating the FreeRDP library from the official repo, please update the following in this file (README.md):
+### ❗When updating the FreeRDP library from the official repo, please update the following table in this file (README.md):
 
-|**Original repository tag/branch used:**| `2.5.0` |
+|**Original repository tag/branch used:**| `2.11.2` |
 | --- | --- |
-|**Original corresponding commit hash:**| `d50aef95520df4216c638495a6049125c00742cb` |
+|**Original corresponding commit hash:**| `a38c1be9eee39a9bc22b511fffe96e63fdf8ebe7` |
 
+#### Updating the fork is done on a dev machine and involves rebasing our changes on top of the new tag/branch from the original repo and **force pushing** the `uipath` branch.
+
+❗The state before the update must be saved in a support branch, named `robot/support/before_update_to_<NEW BASE GIT TAG>`, where `<NEW BASE GIT TAG>` is the tag from the original repo onto which we'll rebase our changes.
+
+The tags from the original repo must be fetched from the original repo and pushed to our fork:
+```pwsh
+git fetch --tags --all
+git push --tags
+```
+
+A work branch should be created from `uipath` and used in the validation phase. It's name would be `feat/update_to_<NEW BASE GIT TAG>`.
+After checking out the branch, the rebase should be performed using the following command:
+```pwsh
+git rebase --onto <NEW BASE Git Tag> <Our 1st customization Commit's Hash> <The work branch>
```

Comment:
@daniel-stanciu, any thoughts on the following?:

The `git rebase --onto` command causes the currently checked out branch to be reset to whatever the 3 arguments dictate. `<The work branch>` could very well be replaced with `uipath`, since it's assumed they point to the same thing.